dev/CommonStyles/InkToolbar_themeresources.xaml

```diff
@@ -411,7 +500,8 @@
                                 BorderBrush="{TemplateBinding BorderBrush}"
                                 BorderThickness="{TemplateBinding BorderThickness}"
                                 Grid.RowSpan="2" Grid.ColumnSpan="2"
-                                Margin="1,0"/>
+                                Margin="1,0"
+                                CornerRadius="{TemplateBinding CornerRadius}"/>
 
                         </Grid>
                         <Rectangle x:Name="SelectionAccent" Style="{StaticResource InkToolbarSelectionAccentStyle}" />
```

Comment:
I think currently, we don't need ExtensionGlyph any more. Is that correct? If yes, we should remove all of them in the code.